### PR TITLE
strip indentation in `not` quickpick

### PR DESCRIPTION
**Pull request by zardoy**
Before:

Screenshot 2022-09-22 at 20 40 20
After:
Screenshot 2022-09-22 at 20 41 10

### CODE REVIEW

File: src/utils/multiline-expressions.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@ import * as vsc from 'vscode'
 export const AllTabs = /^\t+$/
 export const AllSpaces = /^ +$/
 
+export function simpleStripIndent(code: string, joinSep = '\n') {
+  const reNewLine = /\r?\n/
+  return code.split(reNewLine).map((line) => line.replace(/^\s*/, '')).join(joinSep)
+}
```
Comment (ipatalas) [inline src/utils/multiline-expressions.ts]: What's the benefit of that over simple `.replace(/\s+/g, ' ')`?

Comment (zardoy) [reply src/utils/multiline-expressions.ts]:
```suggestion
export function simpleStripIndent(code: string) {
  const reNewLine = /\r?\n/
  return code.replace(/\s+/g, ' '))
}
```

Comment (zardoy) [reply src/utils/multiline-expressions.ts]: Yeah, I like it

Comment (ipatalas) [reply src/utils/multiline-expressions.ts]: Maybe also rename to `stripExcessiveWhitespace`?

**Review by ipatalas — APPROVED** [review]

**Pull request merged**